server/src/main/java/org/apache/kafka/server/config/Defaults.java

```diff
@@ -0,0 +1,278 @@
+    public static final int LOG_CLEANER_IO_BUFFER_SIZE = 512 * 1024;
+    public static final double LOG_CLEANER_DEDUPE_BUFFER_LOAD_FACTOR = 0.9d;
+    public static final int LOG_CLEANER_BACKOFF_MS = 15 * 1000;
+    public static final boolean LOG_CLEANER_ENABLE = true;
+    public static final int LOG_FLUSH_OFFSET_CHECKPOINT_INTERVAL_MS = 60000;
+    public static final int LOG_FLUSH_START_OFFSET_CHECKPOINT_INTERVAL_MS = 60000;
+    public static final int NUM_RECOVERY_THREADS_PER_DATA_DIR = 1;
+    public static final boolean AUTO_CREATE_TOPICS_ENABLE = true;
+
+    /** ********* Replication configuration *********/
+    public static final int CONTROLLER_SOCKET_TIMEOUT_MS = REQUEST_TIMEOUT_MS;
+    public static final int REPLICATION_FACTOR = 1;
+    public static final long REPLICA_LAG_TIME_MAX_MS = 30000L;
+    public static final int REPLICA_SOCKET_TIMEOUT_MS = 30 * 1000;
+    public static final int REPLICA_SOCKET_RECEIVE_BUFFER_BYTES = 64 * 1024;
+    public static final int REPLICA_FETCH_MAX_BYTES = 1024 * 1024;
+    public static final int REPLICA_FETCH_WAIT_MAX_MS = 500;
+    public static final int REPLICA_FETCH_MIN_BYTES = 1;
+    public static final int REPLICA_FETCH_RESPONSE_MAX_BYTES = 10 * 1024 * 1024;
+    public static final int NUM_REPLICA_FETCHERS = 1;
+    public static final int REPLICA_FETCH_BACKOFF_MS = 1000;
+    public static final long REPLICA_HIGH_WATERMARK_CHECKPOINT_INTERVAL_MS = 5000L;
+    public static final int FETCH_PURGATORY_PURGE_INTERVAL_REQUESTS = 1000;
+    public static final int PRODUCER_PURGATORY_PURGE_INTERVAL_REQUESTS = 1000;
+    public static final int DELETE_RECORDS_PURGATORY_PURGE_INTERVAL_REQUESTS = 1;
+    public static final boolean AUTO_LEADER_REBALANCE_ENABLE = true;
+    public static final int LEADER_IMBALANCE_PER_BROKER_PERCENTAGE = 10;
+    public static final int LEADER_IMBALANCE_CHECK_INTERVAL_SECONDS = 300;
+    public static final String INTER_BROKER_SECURITY_PROTOCOL = SecurityProtocol.PLAINTEXT.toString();
+    public static final String INTER_BROKER_PROTOCOL_VERSION = MetadataVersion.latestProduction().version();
+
+    /** ********* Controlled shutdown configuration *********/
+    public static final int CONTROLLED_SHUTDOWN_MAX_RETRIES = 3;
+    public static final int CONTROLLED_SHUTDOWN_RETRY_BACKOFF_MS = 5000;
+    public static final boolean CONTROLLED_SHUTDOWN_ENABLE = true;
+
+    /** ********* Group coordinator configuration *********/
+    public static final int GROUP_MIN_SESSION_TIMEOUT_MS = 6000;
+    public static final int GROUP_MAX_SESSION_TIMEOUT_MS = 1800000;
+    public static final int GROUP_INITIAL_REBALANCE_DELAY_MS = 3000;
+    public static final int GROUP_MAX_SIZE = Integer.MAX_VALUE;
+
+    /** ********* New group coordinator configs *********/
+    public static final boolean NEW_GROUP_COORDINATOR_ENABLE = false;
+    public static final List<String> GROUP_COORDINATOR_REBALANCE_PROTOCOLS = Arrays.asList(Group.GroupType.CLASSIC.toString());
```

Comment:
Should we use `Collections.singletonList()` here?

Reply:
yup, updated this